### ISSUE TITLE
Find new home for a11y content

### ISSUE THREAD

**Issue by indirectlylit** [learningequality/kolibri#5945]
[Accessibility resources in the wiki](https://github.com/learningequality/kolibri/wiki/Accessibility-Resources-(Tools)) should be in our developer documentation, perhaps as a page linked from here:
https://kolibri-dev.readthedocs.io/en/develop/manual_testing.html

**Comment by Lewiscowles1986** [comment]
Tackling the disabling of the wiki and the movement of content should be de-coupled. Only a current internal team member can disable the wiki right? So this seems more in-line with moving the content. It allows a PR to include the text `fixes #{issuenumber}` and auto-close off the 400 issues, at-least one of which I now understand to be a joke

**Comment by indirectlylit** [comment]
thanks, I've updated the issue to have a narrower scope

**Comment by indirectlylit** [comment]
deleted wiki page
